spec/models/user_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/236292892-edaade59-0aa8-4832-b602-d053da85ba5f.png)

- [ ] It would be nice if you make test cases for all the custom methods in your models to follow the project requirements :+1: